packages/anvil-embed-frame/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@anvilco/anvil-embed-frame",
-  "version": "1.1.0",
+  "version": "2.0.0-alpha.0",
```

Comment:
good to move off the alpha version?

Reply:
probably waiting for `lerna publish` to handle it after merging?

Reply:
I will publish both repos after merge!